tests/python/macros/run.mac

```diff
@@ -0,0 +1,18 @@
+/RMG/Geometry/RegisterDetector Germanium germanium 001
+
+/run/initialize
+
+/RMG/Generator/Confine Volume
+/RMG/Generator/Confinement/Geometrical/AddSolid Sphere
+/RMG/Generator/Confinement/Geometrical/Sphere/OuterRadius 1
+
+/RMG/Generator/Confinement/Geometrical/CenterPositionX 0
+/RMG/Generator/Confinement/Geometrical/CenterPositionY 0
+/RMG/Generator/Confinement/Geometrical/CenterPositionZ 0
+
+/RMG/Generator/Select GPS
+/gps/particle e-
+/gps/ang/type iso
+/gps/energy $ENERGY keV
```

Comment:
still using the "old" syntax...

Reply:
damn... let me fix

Reply:
actually this is just a leftover, the tests are in basics/

I'll delete it in the upcoming pr